src/components/views/settings/CryptographyPanel.tsx

```diff
@@ -62,7 +62,7 @@ export default class CryptographyPanel extends React.Component<IProps, IState> {
         }
 
         let noSendUnverifiedSetting: JSX.Element | undefined;
-        if (SettingsStore.isEnabled("blacklistUnverifiedDevices")) {
+        if (SettingsStore.canSetValue("blacklistUnverifiedDevices", null, SettingLevel.DEVICE)) {
```

Comment:
This line is at `true` if we can set the `blacklistUnverifiedDevices` settings on this device ?

Reply:
Yes. Which makes sense, because that's what's going to happen when you toggle the `SettingsFlag`.